### ISSUE TITLE
Question about the regression problem

### ISSUE THREAD

**Issue by Elenore1997** [princeton-nlp/LM-BFF#25]
Hi, I have a question about the method of regression. In section 4.2 in your paper, why use kl-divergence loss between p(yu | xin) and the scaled score (y−vl)/(vu−vl) (loss = loss_fct(logits.view(-1, 2), labels)), but not cross entropy loss? The logits and labels here are both probability distribution on the 2 polarities. 
Thanks in advance!

**Comment by ajfisch** [comment]
Thanks for the question. It should be possible to use either one.

**Comment by Elenore1997** [comment]
> Thanks for the question. It should be possible to use either one.

Thanks for the quick reply!
